### PR TITLE
docs: add docs for BC fusion

### PR DESCRIPTION
**Pull request by forcodedancing**
Add documents for BC fusion

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -299,8 +299,42 @@ const sidebars = {
             label: 'opBNB',
             href:'https://docs.bnbchain.org/opbnb-docs/',
           },
-         
-    ],
+
+          {
+              type: 'category',
```
Comment (zjubfd) [inline sidebars.js:304]: The layout seems wrong, after I run the doc site, I dont know where to go to the pages?

File: docs/bcfusion/overview.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+![img](../assets/bcfusion/phases.png)
+
+There will be several pahses to retrie Beacon Chain:
+
+- **First Sunset Fork** - Some types of Beacon chain transactions will be disabled, for example, TimeLockMsg,
+  TimeRelockMsg,
+  FreezeMsg, IssueMsg, MintMsg, IssueMiniMsg, HTLTMsg, DepositHTLTMsg, MsgCreateValidatorOpen,
+  MsgCreateSideChainValidator, MsgCreateSideChainValidatorWithVoteAddr, MsgEditSideChainValidatorWithVoteAddr,
+  MsgSideChainDelegate, MsgSideChainReDelegate. (Estimated time on mainnet: 2024 Feb)
+- **BSC Feynman Hardfork** - Native validators and staking, native goverenance will be enabled on BNB Smart Chain. (
```
Comment (zjubfd) [inline docs/bcfusion/overview.md:21]: + `The BSC validator/delegator can start migration after the Feynman upgrade.`

**Review by zjubfd — APPROVED** [review]

File: docs/bcfusion/overview.md (hunk 2 above)
Comment (forcodedancing) [reply docs/bcfusion/overview.md]: updated

**Review by Joselinosko — APPROVED** [review]

**Comment by goodgollyholly** [comment]
[](url)

**Pull request merged**